app/interfaces/TeamStandingStatsDTO.java

```diff
@@ -0,0 +1,69 @@
+package interfaces;
+
+/**
+ * TeamStatsDTO contains all of the information necessary for the stats array list
+ *
+ * @author jshyo
+ *
+ */
+public class TeamStandingStatsDTO extends TeamDTO implements Comparable{
+    // Display stats
+    private int wins;
+    private int losses;
+    private int ties;
+    private float winPct;
+    private float PPG;
+    private float PAPG;
+    private int pow;
```

Comment:
We should name these "numPowers", "numBase", "numNegs" for clarity. This DTO should also have an array of tossup values similar to MatchTeamStatsDTO, but as a sum instead of match specific